Formula/curlcpp.rb

```diff
@@ -19,7 +19,7 @@ class Curlcpp < Formula
   uses_from_macos "curl"
 
   def install
-    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_SHARED_LIBS=ON"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DBUILD_SHARED_LIBS=SHARED"
```
Upstream usage of `BUILD_SHARED_LIBS` is non-standard.